instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -108,6 +108,26 @@
 }
 
 
+class MyHooks(object):
+    done = False
+
+    def on_gc_minor(self, stats):
+        print(
```

Comment:
Why `print`? Are you trying to log something? If so, use a proper logger.

Reply:
@ocelotl , it's not required but if anything needs to log will add it accordingly.